lib/ansible/module_utils/urls.py

```diff
@@ -550,7 +550,10 @@ def build_ssl_validation_error(hostname, port, paths):
     msg.append('You can use validate_certs=False if you do'
                ' not need to confirm the servers identity but this is'
                ' unsafe and not recommended.'
-               ' Paths checked for this platform: %s')
+               ' Paths checked for this platform: %s.')
+
+    if exc:
+        msg.append('The exception msg was: %s.' % exc)
 
     raise SSLValidationError(' '.join(msg) % (hostname, port, ", ".join(paths)))
```

Comment:
Yeah -- just for info, to_native() is preferable because, with default arguments, it won't traceback if non-ascii values end up in the exception message.  In some cases, non-ascii will traceback with str() and with ```'%s' % e```.